atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -633,5 +638,30 @@ def run(self):
         self.log.debug("Build information: %s",
                        json.dumps(build_info, sort_keys=True, indent=4))
 
+        # Tag the build
+        if build_id is not None and self.target is not None:
+            self.log.debug("Finding build tag for target %s", self.target)
+            target_info = session.getBuildTarget(self.target)
+            build_tag = target_info['build_tag']
+            self.log.info("Tagging build with %s", build_tag)
+            task_id = session.tagBuild(build_tag, build_id)
+            self.log.debug("Watching task %r", task_id)
+            last = None
+            while last not in ['CLOSED', 'CANCELED', 'FAILED']:
```

Comment:
Yes, I was thinking of adding exactly that, and making sure to package koji_utils.py into the 'koji' sub-package in the spec file.

OK, authorization as well, noted.